### PR TITLE
fix: add billing information in order view page

### LINKED ISSUE #3295: Billing Information not being displayed in the orders
**Describe the bug**

After the billing information was made compulsory, it should be shown in the paid orders like it used to show.

ex : 
image

Currently it is being show like this : 
image

**Additional context**

This would require changes on component level, I will refactor the component according to ES6 standards while working on it

### PR DESCRIPTION
**Pull request by uds5501**
Fixes #3295 

#### Short description of what this resolves:
It implements usage of shows billing details for paid orders.

#### Changes proposed in this pull request:

- Refactor `attendee-list` component according to ES6 standards
- Use billing details part of templates for new orders.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by uds5501** [comment]
Screenshot from 2019-07-19 20-39-42

@shreyanshdwivedi @mrsaicharan1 @niranjan94 @CosmicCoder96 
I had a doubt. What to do in the cases of old orders where billing was not enabled but they were paid anyway. Should I add `company` part of the data in the computed property as a necessity to show this part of the template (and if this is missing, simply display "This is an old order with no billing information" ?)

File: app/components/forms/orders/attendee-list.js [hunk 1]
```diff
@@ -2,20 +2,24 @@ import Component from '@ember/component';
...
+  get buyer() {
+    return this.data.user;
+  }
+
+  @computed('data.user')
+  get holders() {
+    return this.data.attendees;
+  }
+
+  @computed('data.amount', 'data.isBillingEnabled')
```
Comment (kushthedude) [inline app/components/forms/orders/attendee-list.js]: use or

Comment (uds5501) [reply app/components/forms/orders/attendee-list.js]: did it.

Comment (kushthedude) [reply app/components/forms/orders/attendee-list.js]: No, Use like this 
```js
@or('data.amount','data,isBillingEnabled')
showBillingInfo;
``

File: app/components/forms/orders/attendee-list.js [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
-    return true;
-  }),
-  allFields: computed('fields', function() {
+export default class extends Component {
+  @computed('data.user')
+  get buyer() {
+    return this.data.user;
+  }
+
+  @computed('data.user')
```
Comment (niranjan94) [inline app/components/forms/orders/attendee-list.js]: Should be `data.attendees`

File: app/components/forms/orders/attendee-list.js [hunk 3]
```diff
@@ -1,21 +1,26 @@
...
+  get holders() {
+    return this.data.attendees;
+  }
+
+  @computed('data.amount', 'data.isBillingEnabled')
+  get showBillingInfo() {
+    return or('data.amount', 'data.isBillingEnabled');
+  }
+
+  @computed('fields')
```
Comment (niranjan94) [inline app/components/forms/orders/attendee-list.js]: fields is an array. should be `fields.@each.form`

File: app/components/forms/orders/attendee-list.js (hunk 1 above)
Comment (uds5501) [reply app/components/forms/orders/attendee-list.js]: @kushthedude it doesn't work this way.

Comment (niranjan94) [reply app/components/forms/orders/attendee-list.js]: @kushthedude is right. Thats the right way to do it.
If it doesn't work, maybe you're doing something wrong. You can push the code.

Comment (uds5501) [reply app/components/forms/orders/attendee-list.js]: @niranjan94 Alright, taking a look.

**Comment by uds5501** [comment]
@kushthedude @niranjan94 I have pushed the code with correct implementation. The thing which i was doing wrong at my part was using `get` with the or decorator.

File: app/components/forms/orders/attendee-list.js [hunk 4]
```diff
@@ -1,21 +1,24 @@
...
+  get holders() {
+    return this.data.attendees;
+  }
+
+  @or('data.amount', 'data.isBillingEnabled')
+  showBillingInfo;
+
+  @computed('fields.@each.form')
+  get allFields() {
     return groupBy(this.fields.toArray(), field => field.get('form'));
```
Comment (kushthedude) [inline app/components/forms/orders/attendee-list.js]: Use ES6 Getter

**Comment by uds5501** [comment]
@mrsaicharan1 @shreyanshdwivedi Please review

**Comment by uds5501** [comment]
@Anupam-dagar It's passing now.

@shreyanshdwivedi @mrsaicharan1 Please review

**Pull request merged**